jenkins-scripts/docker/ign_msgs-compilation.bash

```diff
@@ -17,7 +17,7 @@ fi
 export BUILDING_SOFTWARE_DIRECTORY="ign-msgs"
 export BUILDING_PKG_DEPENDENCIES_VAR_NAME="IGN_MSGS_DEPENDENCIES"
 # skip ruby on Xenial, because protobuf requires Ruby version >= 2.4.4
-export DOCKER_POSTINSTALL_HOOK="gem install protobuf || if [ `lsb_release -sr` = "16.04" ]; then true; else false; fi"
+export DOCKER_POSTINSTALL_HOOK="(gem install activesupport -v 6.1.4.4 && gem install protobuf) || if [ `lsb_release -sr` = "16.04" ]; then true; else false; fi"
```

Comment:
I think we can even remove the logic for Xenial, which we probably overlooked on #462. What do you think @j-rivero ?

Reply:
Isn't https://build.osrfoundation.org/computer/linux-r2d2.nv.xenial running Xenial? 

The ign_msgs job didn't fail for that node because it seems it's running Xenial (although the docker says Bionic all over the place, probably something else is going there) 
https://build.osrfoundation.org/job/ignition_msgs-ci-pr_any-ubuntu_auto-amd64/735

Reply:
The node is running Xenial as base system but everything is running on Docker this is probably why you are seeing discrepancies between distributions.